pip/download.py

```diff
@@ -593,6 +556,11 @@ def resp_read(chunk_size):
                     break
                 yield chunk
 
+    def written_chunks(chunks):
```

Comment:
The naming of this is a bit weird, this function isn't written chunks, it's something that writes (and yields) chunks.